FusionIIIT/applications/hr2/models.py

```diff
@@ -0,0 +1,108 @@
+
+from django.db import models
+from applications.globals.models import ExtraInfo
+
+
+
+
+class Constants:
+    GENDER_CHOICES = (
+        ('M', 'Male'),
+        ('F', 'Female'),
+          ('O', 'Other'),
+    )
+    DEPARTMENT = (
+        ('CSE', 'CSE'),
+        ('ME', 'Mechanical'),
+        ('ECE', 'ECE'),
+        ('DESIGN', 'DESIGN'),
+    )
+    CATEGORY = (
+        ('SC', 'SC'),
+        ('ST', 'ST'),
+        ('OBC', 'OBC'),
+        ('GENERAL', 'GENERAL'),
+        ('PWD', 'PWD'),
+        
+    )
+    MARITIAL_STATUS = (
+        ('MARRIED', 'MARRIED'),
+        ('UN-MARRIED', 'UN-MARRIED'),
+        ('WIDOW', 'WIDOW'),
+
+    )
+
+    BLOOD_GROUP = (
+        ('AB+', 'AB+'),
+        ('O+', 'O+'),
+        ('AB-', 'AB-'),
+        ('B+', 'B+'),
+        ('B-', 'B-'),
+        ('O-', 'O-'),
+        ('A+', 'A+'),
+        ('A-', 'A-'),
+        
+
+    )
+    
+
+
+
+
+
+# table for employee details
+# Employee model
+class Employee(models.Model):
+
+    extra_info = models.OneToOneField(ExtraInfo, on_delete=models.CASCADE)
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
User is already present in ExtraInfo. Remove it as it is redundant.
Docstrings still not added. Add docstring to every models and views.